firmware/powerboard2/code/app/RTOS/RTOS_componentSpecific.c

```diff
@@ -47,5 +49,6 @@ void RTOS_run100ms(void)
 
 void RTOS_run1000ms(void)
 {
-
+    uint16_t adc = ADC_getChannelData(0);
```

Comment:
Remove this before merging.

Reply:
How are you planning tonight designing the periodic tasks. I was thinking we can use a list of pointers to the the variables, that way we can just iterate though the pointers and publish the data sequentially. 

Also will one message be sent every 1s or all of them?

Reply:
Who are you asking?